site2/website-next/src/pages/community.js

```diff
@@ -521,7 +521,7 @@ export default function Community(props) {
         <section id="community" className="py-12 scrollable">
           <div className="inner">
             <h2 className="text--center">Meet the Community</h2>
-            <p>Pulsar community consists of PMC members and committers. </p>
+            <p>Pulsar community consists of PMC members, committers and contributors. </p>
```

Comment:
Given the roles section of the Apache website, https://www.apache.org/foundation/how-it-works.html#roles, does it make sense to say "Pulsar Development Community" here? My main point is that users are part of the community.

Reply:
That's true. The "Meet the community" page isn't aligned. I'd suggest to create a separate issue to deal with the content on the community page.